code/modules/events/event_dynamic.dm

```diff
@@ -99,6 +99,8 @@ var/list/event_last_fired = list()
 	//and start working via the constructor.
 	new picked_event
 
+	score["eventsendured"]++
```

Comment:
Even the events that don't even impact the round harmfully get points? This seems silly.